scripts/vqvae_test.py

```diff
@@ -0,0 +1,78 @@
+import logging
+from pathlib import Path
+
+import torch
+from huggingface_hub import hf_hub_download
+from PIL import Image
+from torchvision import transforms as T
+from torchvision.utils import save_image
+
+from muse_maskgit_pytorch.vqvae import VQVAE
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+# where to find the model and the test images
+model_repo = "neggles/vaedump"
+model_subdir = "vit-s-vqgan-f4"
+test_images = ["testimg_1.png", "testimg_2.png"]
+
+# where to save the preprocessed and reconstructed images
+image_dir = Path.cwd().joinpath("temp")
+image_dir.mkdir(exist_ok=True, parents=True)
+
+# image transforms for the VQVAE
+transform_enc = T.Compose([T.Resize(512), T.RandomCrop(256), T.ToTensor()])
+transform_dec = T.Compose([T.ConvertImageDtype(torch.uint8), T.ToPILImage()])
+
+
+def get_save_path(path: Path, append: str) -> Path:
+    return path.with_stem(f"{path.stem}-{append}")
```

Comment:
It seems like `path.with_stem` will only work on python 3.9 and higher, replacing this line with `return path.with_name(f'{path.stem}_{stem}{path.suffix}')` should fix it for older versions.

Reply:
of all things...